packages/headless/src/features/commerce/recommendation/recommendation-slice.ts

```diff
@@ -0,0 +1,32 @@
+import {createReducer} from '@reduxjs/toolkit';
+import {
+  fetchRecommendations,
+  updateRecommendationSlotId,
+} from './recommendation-actions';
+import {getRecommendationV2InitialState} from './recommendation-state';
+
+export const recommendationV2Reducer = createReducer(
```

Comment:
```suggestion
export const recommendationReducer = createReducer(
```

Reply:
This is keeping consistency with listings on naming